pkg/controller/jenkins/jenkins_controller.go

```diff
@@ -166,7 +166,7 @@ func (r *ReconcileJenkins) Reconcile(request reconcile.Request) (reconcile.Resul
 		}
 
 		if log.Debug {
-			logger.V(log.VWarn).Info(fmt.Sprintf("Reconcile loop failed: %+v", err))
+			logger.V(log.VDebug).Info(fmt.Sprintf("Reconcile loop failed: %+v", err))
```

Comment:
please revert this line as it is warn not debug

Reply:
So what it is the purpose of the test ? the two lines are identical

Reply:
can we simplify by removing the test here
if log.Debug {
			logger.V(log.VWarn).Info(fmt.Sprintf("Reconcile loop failed: %+v", err))
		} else if err.Error() != fmt.Sprintf("Operation cannot be fulfilled on jenkins.jenkins.io \"%s\": the object has been modified; please apply your changes to the latest version and try again", request.Name) {
			logger.V(log.VWarn).Info(fmt.Sprintf("Reconcile loop failed: %s", err))
		}

Reply:
Seems that in most all cases we log the same thing
logger.V(log.VWarn).Info(fmt.Sprintf("Reconcile loop failed: %s", err))

Reply:
We can't remove the if here, please look at `%+v` vs `%s`. If debug is enabled full stack trace will be printed.

Reply:
ok I see thanks

Reply:
```suggestion
			logger.V(log.VWarn).Info(fmt.Sprintf("Reconcile loop failed: %+v", err))
```

Reply:
fixed